src/olympia/promoted/utils.py

```diff
@@ -0,0 +1,49 @@
+import stripe
+
+from django.conf import settings
+
+from olympia.amo.templatetags.jinja_helpers import absolutify
+from olympia.amo.urlresolvers import reverse
+from olympia.constants.promoted import SPONSORED, VERIFIED
+
+
+def create_or_retrieve_stripe_checkout_session(subscription, customer_email):
+    if subscription.stripe_session_id:
+        return retrieve_stripe_checkout_session(subscription)
```

Comment:
What happens if this doesn't return a session? I.e., if `stripe.checkout.Session.retrieve(subscription.stripe_session_id)` doesn't return a session? Would we want to go ahead and create a new one in that case?

Reply:
It should throw an error but I'll check again

Reply:
So I read the stripe docs again and they say:

> We recommend creating a new Session each time your customer attempts to pay.

I didn't see this the first time so I guess we should only have a `create_stripe_checkout_session` here.